test/uploads.js

```diff
@@ -1,41 +1,38 @@
 require('es6-promise').polyfill()
 
 var should = require('should')
-var strava = require('../')
+const strava = require('../')
 
-describe.skip('uploads_test', function () {
```

Comment:
Re-enabled this test by having it generate a unique timestamp in GPX file for each run.
It does mean the test account will get a lot of fake activities, but without being able to delete them through Strava API, there is not much other choice.

Reply:
The solution I've been working on is to move more tests over to mocks. This makes it easier for casual contributors to run the tests, and also makes it reasonable to run tests in CI. The downside is that the `mocks` can become out of sync with the actual Strava API. But Mocking libraries often have a way to turn themselves into pass-throughs, so it seems it might be possible to have override the mocking behavior in the cases where we do want to run the test suite against a real account. 

Especially now that we can't delete via API, I can imagine few people want to run the test suite against a personal account, or bother with an alt account for this purpose.